publish-packages.sh

```diff
@@ -30,10 +30,10 @@ $working_tree_root/Tools/corerun $working_tree_root/Tools/MSBuild.exe $working_t
 
 if [ $? -ne 0 ]; then
     echo -e "\nAn error occurred. Aborting publish-packages.sh ." >> $publish_packages_log
-    echo "ERROR: An error occurred while building packages, see $publish_packages_log for more details."
+    echo "ERROR: An error occurred while publishing packages, see $publish_packages_log for more details."
```

Comment:
I think these same edits are in #1119, so if #1117 goes first you may need a rebase.